alvr/client_opengl/cpp/graphics.cpp

```diff
@@ -820,22 +836,28 @@ void renderLobbyNative(const EyeInput eyeInputs[2]) {
         g_ctx.hudTextureBitmap.clear();
     }
 
-    ovrRenderer_RenderFrame(g_ctx.lobbyRenderer.get(), eyeInputs, true);
+    ovrRenderer_RenderFrame(g_ctx.lobbyRenderer.get(), eyeInputs, true, false);
 }
 
 void renderStreamNative(void *streamHardwareBuffer, const unsigned int swapchainIndices[2]) {
+
+    EyeInput eyeInputs[2] = {};
+    eyeInputs[0].swapchainIndex = swapchainIndices[0];
+    eyeInputs[1].swapchainIndex = swapchainIndices[1];
+
+    if (streamHardwareBuffer == 0) {
+      ovrRenderer_RenderFrame(g_ctx.streamRenderer.get(), eyeInputs, false, true);
+      return;
+    }
+
```

Comment:
I would make only one call to `ovrRenderer_RenderFrame`, with the edits explained above.